app/services/dashtag/api_service.rb

```diff
@@ -61,8 +63,9 @@ def pull_instagram_posts_from_users_and_parse(user_id)
         GramParser.parse(response.parsed_response)
       end
 
-      def pull_twitter_posts_and_parse(hashtag)
-        response = HTTParty.get("https://api.twitter.com/1.1/search/tweets.json?q=%23#{hashtag}",
+      def pull_twitter_posts_and_parse(hashtags)
+        hashtag_query = hashtags.map { |hashtag| "%23#{hashtag}" }.join("%20AND%20")
+        response = HTTParty.get("https://api.twitter.com/1.1/search/tweets.json?q=#{hashtag_query}",
         :headers => { "Authorization" => "Bearer #{twitter_bearer_token}",
```

Comment:
Oh never mind! It is defined! Thanks for your work! I'll bring your code back :)